core/src/gl/shaderProgram.cpp

```diff
@@ -215,43 +223,76 @@ GLuint ShaderProgram::makeCompiledShader(const std::string& _src, GLenum _type)
 
 }
 
-void ShaderProgram::applySourceBlocks(std::string& _vertSrcOut, std::string& _fragSrcOut) {
+static const std::regex pragmaLine("^\\s*#pragma tangram:\\s+(\\w+).*$");
```

Comment:
this can be a local static const in `ShaderProgram::applySourceBlocks`. I always prefer not to have global variables wherever possible.